nixos/modules/virtualisation/google-compute-image.nix

```diff
@@ -253,11 +253,12 @@ in
               mv -f $SSH_HOST_KEYS_DIR/ssh_host_ed25519_key* /etc/ssh/
               chmod 600 /etc/ssh/ssh_host_ed25519_key
               chmod 644 /etc/ssh/ssh_host_ed25519_key.pub
+              rm $SSH_HOST_KEYS_DIR/*
           else
               echo "Setup of ssh host keys from http://metadata.google.internal/computeMetadata/v1/instance/attributes/ failed."
               false
           fi
-          rm -f $SSH_HOST_KEYS_DIR
+          rm -r $SSH_HOST_KEYS_DIR
```

Comment:
let's make this `rm -rf` to prevent error messages and prompts.

Reply:
@xeji yeah I kinda try avoiding rm -rf's in general in case something unexpected happens but this should be fine so I'll change it. Thanks